### ISSUE TITLE
404 page

### ISSUE THREAD

**Issue by charlax** [jeblister/kube#23]
Thanks for building Kube!

The page on [Hugo themes](https://themes.gohugo.io/kube/) mentions a "404 page : 404 page with animated background" - yet I can't really find it in the code, or on the example website.

I was wondering if you had any plan or recommendation to build such a page.

Thanks.

**Comment by scotte** [comment]
Indeed, it does not appear to exist - the theme includes only an empty 404.html - https://github.com/jeblister/kube/blob/master/layouts/404.html so I had to implement my own.
